dev/PagerControl/PagerControl_themeresources.xaml

```diff
@@ -19,6 +19,73 @@
 
         <x:Double x:Key="PagerControlNumberPanelButtonWidth">32</x:Double>
 
+    <Style x:Key="PagerControlPageNavigationButtonStyle" TargetType="Button">
+        <Setter Property="Background" Value="Transparent"/>
```

Comment:
We could use a `PagerControlPageNavigationButtonBackground` theme resource for this (to match resources like `PagerControlPageNavigationButtonBackgroundPointerOver` if added).

Reply:
Not really a fan since you could easily swap out the style (e.g. using BasedOn), but added it for the sake of it.

Reply:
It matches existing styling APIs we have in UWP/WinUI like [default button styling](https://github.com/microsoft/microsoft-ui-xaml/blob/master/dev/CommonStyles/Button_themeresources.xaml#L125) and thus provides a familiar styling experience. It also enables customers to just use one set of theme resources instead of also having to create a style to complete their button background theming. They can easily copy them from the themeresources file into their app and just edit the color values. No need to start typing up a based on style. 

Customers can also modify these resources once, put them into their app resource dictionary and have them applied *across the app*. If they were to use a BasedOn style instead, they would also have to create a new (implicit) PagerControl style consuming their modified button style (if they have want their multiple PagerControls to automatically have the same page navigation button background, instead of having to manually set the style for every PagerControl instance). Meaning more work when all they want to do is style the background of the PagerControl page navigation buttons.

It will also help with apps like the Fluent XAML Theme Editor (FXTE) which basically gives you a list of customized resources (like background and foreground theme resources) you can copy into your app. If FXTE wouldn't have a background theming resource to override for a customized PagerControl navigation button, then it would, as mentioned above, have to create a new style based on the PagerControlPageNavigationButtonStyle style. For that style to "just work" when copying the FXTE output into your app resources, the editor would also have to create a new implicit PagerControl style. That's more work for the editor here instead of just generating a list of overriden theme resources.

Reply:
In a lot of cases, theme resources make sense, e.g. for controls that are used very often or controls that are hidden inside templates, like the back button of the NavigationView. Having to retemplate just to adjust the background is overkill, having to set background on every button inside your app is also a lot of work. However in the case of PagerControl there is an API to switch the style of those buttons and PagerControl is not a control that will occur dozens of times inside an app. It's a navigation control and those tend to exist a few times often only once inside an app. Of course a resource makes it easier to customize, but it's already quite easy to customize compared to other areas where we use theme resources. But that's just my opinion here.

Reply:
Since this resource will potentially keep customers from having to create two styles (one button style and one PagerControl style (if they happen to have more than one PagerControl)) I consider this a beneficial addition even with the existence of the styling APIs in the Pager Control (especially as it doesn't cost us much maintenance work here thanks to being reviewed before the control will ship (i.e. shipping with a correct/consistent name).

And since you have added the resource anyway, we are all good here :)

Reply:
After discussing this with @Felix-Dev further, we found out that you can actually do this:

```xaml
<Style x:Key="PagerControlPageNavigationButtonStyle" TargetType="Button" BasedOn="{StaticResource PagerControlPageNavigationButtonStyle}">
    <Setter Property="Background" Value="Red" />
</Style>
```

to achieve the same. Given that just swapping the background for the buttons doesn't look that well either and needs more then a color switch, I removed the lightweight styling resource. With almost no effort, devs can still change the background for all buttons so there is little benefit to the resource.